source/includes/_instruments.md

```diff
@@ -327,3 +327,165 @@ Http Status:
 | 400 Bad Request  | The request was malformed.  See response body    | None       | n/a              |
 | 401 Unauthorized | No auth provided, auth failed, or not authorized | None       | n/a              |
 | 404 Not Found    | No allocation with this UUID exists              | None       | n/a              |
+
+
+
+## Instrument Reporting
+
+The [`POST /instruments/reporting`](/#settlement-equity-post-instruments-reporting) endpoint allows platforms to report their system's view on equity ownership for reconcillation purposes.
+
+Equity ownership can be communicated to Goji as a snapshot via multiple calls to [`POST /instruments/reporting`](/#settlement-equity-post-instruments-reporting) describing investor holdings on a per-instrument basis. 
+
+⚠️ All instruments that are part of the same snapshot should share the same `reportId`.
+
+
+## Instrument Reporting Model
+
+### Required Attributes
+
+All of the following attributes of the Instrument Reporting Model are considered 'required' attributes. 
+
+⚠️ In the case of an instrument that has not yet been issued or traded to investors - the holdings section can be left empty.
+
+
+| Key                  | JSON Type | Value Type       | Value Description                                                                                                      |
+|----------------------|-----------|------------------|------------------------------------------------------------------------------------------------------------------------|
+| reportId             | String    | UUID             | A platform-generated UUID for this equity ownership snapshot, shared between all instruments included in this snapshot.|
+| instrumentSymbol     | String    | InstrumentSymbol | The unique ID/symbol of the instrument registered by the platform.                                                     |
+| timestamp            | String    | ISO 8601         | The time at which the snapshot of all instruments' holdings was originally taken.                                      |
+| holdings[]           | Array     | Object           | The list of investor holdings in an `instrumentSymbol`.                                                                |
+| holdings[].clientId  | String    | Object           | The clientId of the investor associated with this individual holding.                                                  |
+| holdings[].quantity  | Number    | Number           | The number of shares owned by the `clientId`.                                                                          |
+
+
+## `POST /instruments/reporting`
+
+```http
+POST /instruments/reporting HTTP/1.1
+Host: api-sandbox.goji.investments
+Content-Type: application/json
+Authorization: Basic ...
+X-GOJI-CLIENT-ID: 59425d3d-cf73-44ff-aecb-590cd198a4bc
+
+{
+  "reportId": "68f54225-6760-449d-8f38-a003d1dc535a",
+  "instrumentSymbol": "SPV1",
+  "timestamp": "2020-01-09T08:53:20.963Z",
```

Comment:
This worth updating assuming my previous comment is correct?

Reply:
sure